Dockerfile

```diff
@@ -4,9 +4,6 @@ FROM --platform=${BUILDPLATFORM:-linux/amd64} node:18 AS ui
 # See for details: https://github.com/hadolint/hadolint/wiki/DL4006
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-ARG REACT_APP_API_ENDPOINT
-ENV REACT_APP_API_ENDPOINT=${REACT_APP_API_ENDPOINT}
```

Comment:
Why did you remove these 2 lines?

Reply:
With these 2 lines, `.env` doesn't work with a container. As the release stage doesn't have this env variable and the value is an empty string during building, I don't see the reason to keep them.

Reply:
As I work on a combined docker compose file, I can keep these lines and update env in the compose file. I still think they are not used, but I'd like to delegate to a frontend developer to decide.

Reply:
For sure, it is better to decide on the Frontend side. I'm also not 100% sure that this var is needed, but since it was there before (even my) changes to the Dockerile, I'd prefer it to be kept until we have a confirmation from a Frontend that it is not needed for any possible use case.

As far as I see, .env can contain whatever variables you want. If a var is not there, it will be empty, if there is a var, not referenced in the compose or Dockerfile, it will be ignored.

This particular var is not for .env and compose. It is for building the frontend inside the build stage on the CI as a 1st prio.

Reply:
Revert this change.